website/src/_posts/2019-08-22-beam-2.15.0.md

```diff
@@ -0,0 +1,71 @@
+---
+
+layout: post
+title:  "Apache Beam 2.15.0"
+date:   2019-08-14 00:00:01 -0800
+# Date above corrected but keep the old URL:
+permalink: /blog/2019/08/14/beam-2.15.0.html
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+        - yifanzou
+
+ ---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+ We are happy to present the new 2.15.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#2130-2019-08-14) for this release.<!--more-->
+For more information on changes in 2.15.0, check out the
+[detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12345489).
+
+ ## Highlights
```

Comment:
Can you make a pass over the JIRA list and see what other interesting things are there?

Reply:
The release note contains all JIRAs is listed above.